changelogs/fragments/20250103-sanity-fixes.yaml

```diff
@@ -0,0 +1,25 @@
+---
+minor_changes:
+  - ec2_customer_gateway - Fix documentation for the return block (https://github.com/ansible-collections/community.aws/pull/2354).
+breaking_changes:
+  - cloudfront_invalidation - Return value key ``items`` in ``invalidation -> invalidation_batch -> paths`` has been renamed to ``elements``.
```

Comment:
The steps are in general: first deprecate an option and then remove it once the deprecation period has passed.